pkg/controller/container-runtime-config/helpers.go

```diff
@@ -431,14 +435,18 @@ func validateUserContainerRuntimeConfig(cfg *mcfgv1.ContainerRuntimeConfig) erro
 	}
 
 	ctrcfg := cfg.Spec.ContainerRuntimeConfig
-	if ctrcfg.PidsLimit > 0 && ctrcfg.PidsLimit < minPidsLimit {
-		return fmt.Errorf("invalid PidsLimit %q, cannot be less than 20", ctrcfg.PidsLimit)
+	if ctrcfg.PidsLimit != nil && *ctrcfg.PidsLimit > 0 && *ctrcfg.PidsLimit < minPidsLimit {
```

Comment:
This should be `>= 0` as well right?

Reply:
Not really. if it's 0, it is valid, do not need to return the error below.